Resources/Prototypes/Recipes/Lathes/clothing.yml

```diff
@@ -570,6 +570,38 @@
   id: ClothingUniformJumpsuitPyjamaSyndicateRed
   result: ClothingUniformJumpsuitPyjamaSyndicateRed
 
+#Start DeltaV
```

Comment:
move all these recipes to the deltav folder

Reply:
Moved.